public/js/src/api.js

```diff
@@ -0,0 +1,53 @@
+const POLITICS_URL = `https://content.guardianapis.com/search?from-date=${yesterday()}&to-date=${today()}&q=politics&api-key=test&show-fields=all`
+
+const frontPage = new FrontPage();
+
+fetchArticles();
```

Comment:
I like that you've separated your fetching code from your models! I wonder whether you could separate this more cleanly from your interface though by putting the actual code that initiates the fetch into your interface.

Reply:
right, so I think I get what you mean :)

Again just to check I understand correctly, I would still have an `api.js` file, but I would invoke the api request functions from the interface file instead?

Reply:
Yes :)